gabbi/httpclient.py

```diff
@@ -55,21 +54,21 @@ def request(self, absolute_uri, method, body, headers, redirect):
 
 
 class VerboseHttp(Http):
-    """A subclass of Http that verbosely reports on activity.
+    """A subclass of ``Http`` which verbosely reports on activity.
 
     If the output is a tty or ``GABBI_FORCE_COLOR`` is set in the
     environment, then output will be colorized according to ``COLORMAP``.
 
     Output can include request and response headers, request and
-    response body content (if of a printable content-type), or both.
+    response body content (if of a printable content type), or both.
 
     The color of the output has reasonable defaults. These may be overridden
     by setting the following environment variables
 
-    * GABBI_CAPTION_COLOR
-    * GABBI_HEADER_COLOR
-    * GABBI_REQUEST_COLOR
-    * GABBI_STATUS_COLOR
+    * ``GABBI_CAPTION_COLOR``
+    * ``GABBI_HEADER_COLOR``
+    * ``GABBI_REQUEST_COLOR``
+    * ``GABBI_STATUS_COLOR``
 
     to any of: BLACK RED GREEN YELLOW BLUE MAGENTA CYAN WHITE
```

Comment:
why backticks above but not here?

Reply:
Glad you noticed. I figured that'd be overkill here, doing more harm than good.